Formula/nushell.rb

```diff
@@ -4,6 +4,7 @@ class Nushell < Formula
   url "https://github.com/nushell/nushell/archive/0.39.0.tar.gz"
   sha256 "86eb8079b88896dfbfb8e165adba75a7de3477b21f20ee8282efff2f0ab1356b"
   license "MIT"
+  revision 1
```

Comment:
```suggestion
```
If `--workspace` just makes the default explicit then this should have no user-facing changes, right? In which case, we can omit the `revision` bump.

Reply:
Fair enough!